src/client/src/pages/detail/form/location/coordinatesSegment.tsx

```diff
@@ -158,6 +155,7 @@ const CoordinatesSegment: React.FC<CoordinatesSegmentProps> = ({
     const floatValue = parseFloatWithThousandsSeparator(value);
     // verify coordinates are in bounding box
     if (
+      floatValue &&
```

Comment:
FYI: Die Zahl Null `0` evaluiert zu `false`. Daher finde ich es nicht so gut, wenn eine Zahl so in einer condition verwendet wird. Hier ist es wahrscheinlich kein Problem da `0` nicht in der BoundingBox sein kann?